spec/factories/grant.rb

```diff
@@ -2,7 +2,7 @@
   factory :grant do
     name { Faker::Hipster.sentence }
     hidden false
-    max_funding_dollars { 5_000 }
+    funding_levels_csv { 5_000 }
```

Comment:
the funding_levels_csv is a specially-formatted string, see grant.rb:62.  It's a comma-separated list of individual integers or ranges.